### ISSUE TITLE
wrong-type-argument number-or-marker-p with wg-mode-line-add-display

### ISSUE THREAD

**Issue by mohkale** [tlh/workgroups.el#54]
At first, I thought this was because I was using spaceline. But after running in emacs -q I honestly have no idea whats causing this. I'm running on `GNU Emacs 26.1 (build 1, x86_64-w64-mingw32) 2018-05-30`.

I run the following commands in order:
```
emacs -q
M-x toggle-debug-on-error
M-x package-initialize
M-: (require 'workgroups)
M-x workgroups-mode
```

and then I recieve the following stacktrace. 

```
Debugger entered--Lisp error: (wrong-type-argument number-or-marker-p nil)
  wg-mode-line-add-display()
  workgroups-mode(toggle)
  funcall-interactively(workgroups-mode toggle)
  call-interactively(workgroups-mode record nil)
  command-execute(workgroups-mode record)
  execute-extended-command(nil "workgroups-mode" "workgroups-mo")
  funcall-interactively(execute-extended-command nil "workgroups-mode" "workgroups-mo")
  call-interactively(execute-extended-command nil nil)
  command-execute(execute-extended-command)
```

because `wg-mode-line-add-display` is called after everything else is initted, it's not really a problem, in so far as because the mode is activated and works. It's just I don't like recieving a stacktrace or error message every time I activate the mode. 

I've also tried setting `wg-mode-line-on` to nil after requiring workgroups, before initialising the mode. but I still get the stacktrace.

**Comment by qdot** [comment]
@MoHKale This package hasn't been updated in 8 years, and is probably defunct. You might want to try something like https://github.com/wasamasa/eyebrowse or https://github.com/nex3/perspective-el

**Comment by mohkale** [comment]
Yeah... I realised that after the fact, though still thought it merited creating an issue... thnx for the alternatives. Is there any plan to include a section at the top of the README mentioning that and those alternatives. This repo is still the first result when you google "emacs workgroups".

**Comment by rahenry** [comment]
What you suggest would be good. It just depends if the author/maintainer randomly shows up again or not!
